src/main/content/general-references.html

```diff
@@ -1,40 +1,74 @@
 ---
 layout: doc
 title: General Reference
-type: general
 css:
 - commonConfigFeature
 - general-reference
 js: general-references
-permalink: /docs/ref/general/
+permalink: /docs/
 ---
-{% assign generalDocs = site.pages | where: 'layout', 'general-reference' | where: 'type', 'general' | sort: 'title' %}
+{% assign collectionsDoc = site.pages %}
 
-<div id="background-container">
-    <div class="row">
-        <h1 id="docs-title">Kabanero Docs</h1>
-    </div>
+{% assign docCategories = '' | split: '' %}
+
+{% for doc in collectionsDoc | where: doc.doc-category %}
```

Comment:
This line causes casues an warning/error "Expected end_of_string but found pipe in" after doing some reading I believe it's because the for loop statment in jekyll doesnt accept the | where pipe an alternative can be using an if statment instead like so 

{% for doc in collectionsDoc %}
    {% if doc.doc-category %}
        {% assign docCategory = doc.doc-category | split: '/n' %}
        {% assign docCategories = docCategories | concat: docCategory %}
    {% endif %}
{% endfor %}